File: tournaments/alphabeticShift/alphabeticShift.js
function alphabeticShift(inputString) {
    const n = inputString.length;
    const result = new Array(n);
    const aCode = 'a'.charCodeAt();
    let index = 0;

    for (const letter of inputString) {
        result[index++] = String.fromCharCode(aCode + (letter.charCodeAt() - aCode + 1) % 26);
    }

    return result.join('');
}

File: tournaments/alphabeticShift/alphabeticShift.cpp
std::string alphabeticShift(std::string inputString) {
  for (int i = 0; i < inputString.size(); ++i) {
    int number = inputString[i] - 'a';
    number = (number + 1) % 26;
    inputString[i] = 'a' + number;
  }
  return inputString;
}
